test/Winton.Extensions.Configuration.Consul.Test/Extensions/KVPairQueryResultExtensionsTests.cs

```diff
@@ -247,8 +253,9 @@ private void ShouldRemoveSpecifiedKeySection()
                 _parser
                     .Setup(p => p.Parse(It.IsAny<Stream>()))
                     .Returns(new Dictionary<string, string> { { "Key", "Value" } });
+                _source.KeyToRemove = "path";
```

Comment:
I think `ShouldRemoveSpecifiedKeySection` is now (rightly) a test that should just belong in the tests of `KVPair.ConvertToConfig`. Which is [already](https://github.com/wintoncode/Winton.Extensions.Configuration.Consul/blob/master/test/Winton.Extensions.Configuration.Consul.Test/Extensions/KVPairExtensionsTests.cs#L23) covered, so we can delete this test case now.